stylesheets/_typography.scss

```diff
@@ -15,6 +15,8 @@
 
 // CORE FONTS - NEW TRANSPORT
 
+// scss-lint:disable NameFormat
```

Comment:
Is this being disabled so that `_something-` is acceptable? I think there's a provision in the linter to allow hyphenated names that start with an `_`. Though it looks like that's enabled: https://github.com/alphagov/govuk-lint/blob/master/configs/scss_lint/gds-sass-styleguide.yml#L94